sdk/translation/Azure.AI.Translation.Text/samples/Sample2_Translate.md

```diff
@@ -0,0 +1,366 @@
+
+```C#
+try
+{
+    TextTypes textType = TextTypes.Html;
+
+    IEnumerable<string> targetLanguages = new[] { "cs" };
+    IEnumerable<string> inputTextElements = new[]
+    {
+        "<html><body>This <b>is</b> a test.</body></html>"
+    };
+
+    Response<IReadOnlyList<TranslatedTextItem>> response = await client.TranslateAsync(targetLanguages, inputTextElements, textType: textType).ConfigureAwait(false);
+    IReadOnlyList<TranslatedTextItem> translations = response.Value;
+    TranslatedTextElement translation = translations.FirstOrDefault();
+
+    Console.WriteLine($"Detected languages of the input text: {translation?.DetectedLanguage?.Language} with score: {translation?.DetectedLanguage?.Score}.");
+    Console.WriteLine($"Text was translated to: '{translation?.Translations?.FirstOrDefault().To}' and the result is: '{translation?.Translations?.FirstOrDefault()?.Text}'.");
+}
+catch (RequestFailedException exception)
+{
+    Console.WriteLine($"Error Code: {exception.ErrorCode}");
+    Console.WriteLine($"Message: {exception.Message}");
+}
+```
+
+### Don’t translate specific entity name in a text
+It's sometimes useful to exclude specific content from translation. You can use the attribute class=notranslate to specify content that should remain in its original language. In the following example, the content inside the first div element won't be translated, while the content in the second div element will be translated.
+
+```C#
+try
+{
+    TextTypes textType = TextTypes.Html;
+
+    string from = "en";
+    IEnumerable<string> targetLanguages = new[] { "cs" };
+    IEnumerable<string> inputTextElements = new[]
+    {
+        "<div class=\"notranslate\">This will not be translated.</div><div>This will be translated. </div>"
+    };
+
+    Response<IReadOnlyList<TranslatedTextItem>> response = await client.TranslateAsync(targetLanguages, inputTextElements, textType: textType, from: from).ConfigureAwait(false);
+    IReadOnlyList<TranslatedTextItem> translations = response.Value;
+    TranslatedTextElement translation = translations.FirstOrDefault();
+
+    Console.WriteLine($"Detected languages of the input text: {translation?.DetectedLanguage?.Language} with score: {translation?.DetectedLanguage?.Score}.");
+    Console.WriteLine($"Text was translated to: '{translation?.Translations?.FirstOrDefault().To}' and the result is: '{translation?.Translations?.FirstOrDefault()?.Text}'.");
+}
+catch (RequestFailedException exception)
+{
+    Console.WriteLine($"Error Code: {exception.ErrorCode}");
+    Console.WriteLine($"Message: {exception.Message}");
+}
+```
+
+### Translate specific entity name in a text applying a dictionary
+If you already know the translation you want to apply to a word or a phrase, you can supply it as markup within the request. The dynamic dictionary is safe only for compound nouns like proper names and product names.
```

Comment:
I think we need to be more explicit what "safe" means in this context.

Reply:
Taken from the general documentation. I will pass the feedback and we will improve for GA.